wrappers/python/pyrs_device.cpp

```diff
@@ -7,6 +7,8 @@ Copyright(c) 2017 Intel Corporation. All Rights Reserved. */
 #include <librealsense2/hpp/rs_record_playback.hpp> // for downcasts
 #include <common/metadata-helper.h>
 
+#include <iostream>
```

Comment:
is it needed?

Reply:
Not sure, I see it was added specially, so could be:
https://github.com/IntelRealSense/librealsense/commit/87422e1504251f31ae26b17a73f8327dd75a51d0
Better to align do dev.
We can try and remove it there and see if it works later